src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/TrafficSeo/Meta/SetUpUrlType.php

```diff
@@ -52,44 +54,92 @@ class SetUpUrlType extends AbstractType
      */
     private $isHostMode;
 
+    /*
+    * @var Tools
+    */
+    private $tools;
+
     /**
      * SetUpUrlType constructor.
      *
+     * @param \Symfony\Component\Translation\TranslatorInterface $translator
+     * @param array $locales
      * @param array $canonicalUrlChoices
      * @param bool $isHtaccessFileWritable
      * @param bool $isHostMode
+     * @param Tools $tools
      */
     public function __construct(
+        TranslatorInterface $translator,
+        array $locales,
         array $canonicalUrlChoices,
         $isHtaccessFileWritable,
-        $isHostMode
+        $isHostMode,
+        $tools
     ) {
+        parent::__construct($translator, $locales);
         $this->canonicalUrlChoices = $canonicalUrlChoices;
         $this->isHtaccessFileWritable = $isHtaccessFileWritable;
         $this->isHostMode = $isHostMode;
+        $this->tools = $tools;
     }
 
     /**
      * {@inheritdoc}
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        $friendlyUrlHelp = $this->trans(
+            'Enable this option only if your server allows URL rewriting (recommended).',
+            'Admin.Shopparameters.Help'
+        );
+
+        if (!$this->tools->isModRewriteActive()) {
+            $friendlyUrlHelp .=
+                '</br>' . $this->trans(
```

Comment:
I don't have a cleaner way, but here is a suggestion to be W3C compliant 😄
```suggestion
                '<br>' . $this->trans(
```
Or
```suggestion
                '<br/>' . $this->trans(
```